track/templates/en/guidance.html

```diff
@@ -3,13 +3,73 @@
 {% block title %}Read guidance{% endblock %}
 {% block pageid_en %}guidance{% endblock %}
 {% block pageid_fr %}directives{% endblock %}
-{% block description %}Read guidance.{% endblock description %}
+{% block description %}Project guidance.{% endblock description %}
 
 {% block content %}
 <section id="main-content" class="flex-1">
 	<div class="container mx-auto items-center sm:w-4/5 xl:w-3/5 mt-6 mb-6">
-		<h1>Read guidance</h1>
-		<p>Page content here</p>
+		<h1>Project guidance</h1>
+		
+        <ol>
+
+		   <li class="mb-6"> Identify key resources required to act as central point(s) of contact with TBS and the HTTPS Community of Practice.  Establish connections via the GCTools channels at: 
+		
+		  		 <ul>
+
+					 <li>GCconnex: <a href="https://gcconnex.gc.ca/groups/profile/35218846/gc-https-everywhere-2018-https-partout-dans-le-gc-2018">HTTPS Everywhere 2018</a></li>
```

Comment:
if you add `class="text-https-blue hover:text-black font-bold"` to the links, their style which match the rest of the site

Reply:
tx Eva will do this